test/core/support/alloc_test.c

```diff
@@ -48,8 +48,8 @@ static void test_custom_allocs() {
   gpr_allocation_functions fns = {fake_malloc, fake_realloc, fake_free};
 
   gpr_set_allocation_functions(fns);
-  GPR_ASSERT((void *)0xdeadbeef == gpr_malloc(0xdeadbeef));
```

Comment:
Alright, so the warning is: ..\..\..\..\test\core\support\alloc_test.c(51): warning C4312: 'type cast': conversion from 'unsigned int' to 'void *' of greater size [c:\Users\jtattermusch\github\grpc\vsprojects\vcxproj\test\alloc_test\alloc_test.vcxproj]

Basically casting (void *)0xdeadbeef is illegal.